### ISSUE TITLE
Add Undo/Redo support to the Find and Replace windows

### ISSUE THREAD

**Issue by vstepaniuk** [geany/geany#2499]
Add Undo/Redo support to the Find and Replace windows

**Comment by elextr** [comment]
Don't understand.

**Comment by vstepaniuk** [comment]
![Screenshot from 2020-05-09 11-40-17](https://user-images.githubusercontent.com/24701640/81470208-e397b800-91e9-11ea-8b89-4df80c587984.png)

**Comment by vstepaniuk** [comment]
![Screenshot from 2020-05-09 11-42-05](https://user-images.githubusercontent.com/24701640/81470251-2194dc00-91ea-11ea-9d22-57b41b3ad41d.png)

**Comment by elextr** [comment]
Editing of fields in dialog boxes is handled by the GTK toolkit, Geany only sees the end result when one of the buttons is pressed.  So you would probably need to take it up with the GTK project.

**Comment by codebrainz** [comment]
https://gitlab.gnome.org/GNOME/gtk/issues/1690

TLDR; not gonna happen for GTK3 but already implemented in GTK4.
